src/util/app.js

```diff
@@ -27,6 +27,15 @@ export const appQueries = {
             key: SYSTEM_SETTINGS,
         },
     },
+    systemInfo: {
```

Comment:
I think you can get the systemInfo using the [useConfig](https://developers.dhis2.org/docs/app-runtime/hooks/useconfig/) hook - this would make use of the settings already in the provider rather than making an extra API call.

```js
import { useConfig } from '@dhis2/app-runtime'

const { systemInfo } = useConfig()
const { calendar = 'gregory' } = systemInfo
```